library/src/scripts/splash/splashStyles.ts

```diff
@@ -60,12 +60,15 @@ export const splashVariables = useThemeCache(() => {
         borderColor: globalVars.mainColors.fg.fade(0.4),
     });
 
-    const outerBackground: IBackground = makeThemeVars("outerBackground", {
+    const isContrastLight = colors.contrast instanceof ColorHelper && colors.contrast.lightness() >= 0.5;
+    const outerBackground = makeThemeVars("outerBackground", {
         color: colors.primary,
         backgroundPosition: "50% 50%",
         backgroundSize: "cover",
         image: assetUrl("/resources/design/fallbackSplashBackground.svg"),
         fallbackImage: assetUrl("/resources/design/fallbackSplashBackground.svg"),
+        useFilter: false,
```

Comment:
I wouldn't use the word "filder" here because it makes me think of CSS filters: https://developer.mozilla.org/en-US/docs/Web/CSS/filter

I think "overlay" or "colorOverlay" is easier to understand.

Reply:
Implemented. The variables are now `useOverlay` and `overlayColor`. I've also made the overlay its own element instead of a filter.